sysvinit/gentoo/cloud-init-hotplug

```diff
@@ -0,0 +1,18 @@
+#!/sbin/openrc-run
+
+command="/usr/lib/cloud-init/cloud-init-hotplugd"
+command_background=yes
+pidfile="/run/$RC_SVCNAME.pid"
+
+depend() {
+  before cloud-init
```

Comment:
> Why position before cloud-init?

I based this on systemd/cloud-init-hotplugd.socket defining "WantedBy=cloud-init.target" which I took to mean that the hotplug should be run before cloud-init

Reply:
The [cloud-init.target is just a synchronization point in boot that means that all cloud-init related units/services have finished running](https://github.com/canonical/cloud-init/blob/main/systemd/cloud-init.target#L6-L7) It doesn't represent a specific point in time before cloud-init is running, but a point in boot that external services/units can be sure cloud-init's 4 boot stages are all done The `WantedBy=cloud-init.target` in each of our subordinate units/services represented by our separate boot stages basically just registers those stages as blocking the overall status of `cloud-init.target`.  Ultimately, `cloud-init.target` is only completed/unblocked once we ultimately complete running the final boot stage  `cloud-init-final.service` since it's our last ordered systemd unit ordered `After=cloud-config.service. So, I think the equivalent we want here is `after cloud-final` since there is no cloud-init.target represented in gentoo-openrc land

Reply:
Ok, I changed this to "after cloud-final".